api/actions/serializers.py

```diff
@@ -138,7 +138,7 @@ def create(self, validated_data):
                 return target.run_submit(user)
         except InvalidTriggerError as e:
             # Invalid transition from the current state
-            raise Conflict(e.message)
+            raise Conflict(str(e))
```

Comment:
This won't work because of the way `InvalidTriggerError` is implemented.

```python

In [1]: from osf.exceptions import InvalidTriggerError

In [2]: exc = InvalidTriggerError(trigger='foo', state='bar', valid_triggers={'foo'})

In [3]: str(exc)
Out[3]: ''

In [4]: exc.message
Out[4]: 'Cannot trigger "foo" from state "bar". Valid triggers: set([\'foo\'])'
```

So you could fix this by keeping `e.message`. It'd probably be better to instead make sure that `str(e)` returns the message. You can do this by doing

```python
        super(Exception, self).__init__(self.message)
```

in the InvalidTriggerError exception.